lib/utils.js

```diff
@@ -175,8 +256,54 @@ const getMainCodeOwnerfile = async () => {
  */
 const isChangelogLabel = (label) => {
   return label.trim().toUpperCase().startsWith(CHANGELOG_START);
+};
+
+/**
+ * This functions gets the last review of a specific type from a pull request.
+ * For example, if type is CHANGES REQUESTED, this function will return
+ * the last review where the state was CHANGES REQUESTED.
+ *
+ * @param {import('probot').Context} context
+ * @param {string} type
+ */
+const getLastReviewOfSpecificType = async (context, type, pullRequestNumber) => {
+  const {data: allReviews} = await context.github.pulls.listReviews(
+    context.repo({
+      pull_number: pullRequestNumber
+    })
+  );
+  const reviewOfType = allReviews.find(review => {
+    return review.state === type;
+  });
+  console.log({reviewOfType})
+  return reviewOfType;
 }
 
+/**
+ * This function pings and assigns all pending reviewers to a pull request.
+ * @param {import('probot').Context} context
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ * @param {string[]} reviewers
+ */
+const pingAndAssignReviewers = async (context, pullRequest, reviewers) => {
```

Comment:
Suggestion: Change this to `pingAndAssignUsers` and use it everywhere whether we need to assign author or reviewer(s).

Reply:
Why not here itself? Would prevent any extra follow ups since this is the last PR for your M3.

Reply:
Hmmm
Okay. Done.